advanced/myserverextension/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "jupyterlab_myserverextension",
```

Comment:
Similar to the comment above: maybe this could be `@jupyterlab-examples/server-extension`?

Reply:
Thanks for the suggestion